DSC/dsc_to_pif.py

```diff
@@ -0,0 +1,88 @@
+import csv
+import argparse
+from pypif import pif
+from pypif.obj import *
+
+
+def netzsch_3500_to_pif(closed_csv):
+
+    # create chemical system and property array
+    my_pif = ChemicalSystem()
+    my_pif.properties = []
+
+    # Store index so that iteration can start at next row. Default to False when no header is found.
+    header_row_index = False
+
+    # Initialize arrays for heat capacity and conditions (time and temp)
+    temp_array = []
+    time_array = []
+    heat_capacity_array = []
+
+    # open csv, iterate through rows.
+    with open(closed_csv, 'rU') as open_csv:
+        reader = csv.reader(open_csv)
+        for index, row in enumerate(reader):
+
+            # meta data is stored above property with header column = row[0]
+
+            # set values based on row[0]
+            if '#IDENTITY:' in row[0]:
+                my_pif.chemical_formula = row[1].strip()
+
+            if "#INSTRUMENT:" in row[0]:
+                measurement_device = Instrument(name=row[1].split(" ")[0], model=row[1].split(" ")[2])
+
+            if "#SAMPLE MASS /mg:" in row[0]:
+                prop = Property(name='sample mass', scalars=row[1], units='mg')
+                my_pif.properties.append(prop)
+
+            if "#DATE/TIME:" in row[0]:
+                date = Value(name="Experiment date", scalars=row[1].strip())
+
+            if "#TYPE OF CRUCIBLE:" in row[0]:
+                crucible = Value(name="Crucible", scalars=row[1].strip()+" "+row[2].strip())
+
+            if "#PROTECTIVE GAS:" in row[0]:
+                atmosphere = Value(name="Atmosphere", scalars=row[1].strip())
+
+            # Temp indicates header row. Define header_row_index.
+            if '##Temp.' in row[0]:
+                header_row_index = index
+
+            # Iterate through values after header_row.
+            if header_row_index:
+                if index > header_row_index:
+                    temp_array.append(row[0])
+                    time_array.append(row[1])
+                    heat_capacity_array.append(row[2])
+
+        # define property and append scalar values
```

Comment:
You can close the file now, so drop the following lines down one level of indentation.